lib/internal/errors.js

```diff
@@ -946,14 +946,7 @@ E('ERR_INVALID_ARG_TYPE',
       const type = name.includes('.') ? 'property' : 'argument';
       msg += `"${name}" ${type} `;
     }
-
-    // determiner: 'must be' or 'must not be'
-    if (typeof expected === 'string' && expected.startsWith('not ')) {
-      msg += 'must not be ';
-      expected = expected.replace(/^not /, '');
-    } else {
-      msg += 'must be ';
-    }
+    msg += 'must be ';
```

Comment:
Could simplify this further by including the literal inline inside the two conditionals above.

Reply:
I'm not that a big fan of duplication.